test/Container/ApplicationFactoryTest.php

```diff
@@ -1071,4 +1071,226 @@ public function testCanSpecifyRouteOptionsViaConfiguration()
         $this->assertInstanceOf(Route::class, $route);
         $this->assertEquals($expected, $route->getOptions());
     }
+
+    /**
+     * @expectedException \Zend\Expressive\Container\Exception\InvalidArgumentException
```

Comment:
We prefer the "old" `setExpectedException` because:

* We place it just above of the statement which will throw the exception so there is less risk of catch an exception thrown by any other unexpected site.

* Make easy for many tools track the uses of the exception when PHP 5.5 ::class is used too.